### DISCUSSION TITLE
How to transfer settings to another computer

### DISCUSSION THREAD

**Discussion by Erikgobrrr** [ScratchAddons/ScratchAddons#6553] [Help] [answered]
I wanna transfer scratch addons info from one of my computers to another. Doing it manually is a pain
any way to do it?

**Comment by Samq64** [comment, marked as answer]
Open the full settings page, click more settings in the bottom left and then export settings. Once the file is moved to the other computer you can open more settings there, click import settings and choose the file.

Alternatively if you're browser is set to automatically sync everything it should also sync your Scratch Addons settings automatically.
